core/controllers/learner_goals.py

```diff
@@ -55,18 +58,14 @@ class LearnerGoalsHandler(
     }
 
     @acl_decorators.can_access_learner_dashboard
-    def post(self, activity_type: str, topic_id: str) -> None:
+    def post(self, activity_type: str, topic_id: str) -> None:  # pylint: disable=unused-argument
```

Comment:
You'll need a comment to justify why the pylint exception is needed (rather than changing the signature of the handler).

Reply:
Added a comment PTAL!